snap/integrity/dmverity/veritysetup.go

```diff
@@ -81,11 +84,59 @@ func getRootHashFromOutput(output []byte) (rootHash string, err error) {
 	return rootHash, nil
 }
 
+func verityVersion() (major, minor, patch int, err error) {
+	cmd := exec.Command("veritysetup", "--version")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return -1, -1, -1, osutil.OutputErr(output, err)
+	}
+
+	exp := regexp.MustCompile(`(\d+\.)?(\d+\.)?(\*|\d+)`)
```

Comment:
Looks like regex tries to cater for version like "1.2.*" which is not supported at function level. Looks like it can be simplified.